docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -31,17 +31,18 @@ Having defined a partitioned asset or job, you can:
-| <PyObject object="HourlyPartitionsDefinition" />  | A partitions definition with a partition for each hour.                           |
-| <PyObject object="DailyPartitionsDefinition" />   | A partitions definition with a partition for each day.                            |
-| <PyObject object="WeeklyPartitionsDefinition" />  | A partitions definition with a partition for each week.                           |
-| <PyObject object="MonthlyPartitionsDefinition" /> | A partitions definition with a partition for each month.                          |
-| <PyObject object="StaticPartitionsDefinition" />  | A partitions definition with a fixed set of partitions.                           |
-| <PyObject object="MultiPartitionsDefinition" />   | A partitions definition with multiple dimensions.                                 |
-| <PyObject object="MultiPartitionKey" />           | A multi-dimensional partition key.                                                |
-| <PyObject object="DynamicPartitionsDefinition" /> | A partitions definition whose partitions can be dynamically added and removed.    |
+| Name                                                            | Description                                                                       |
+| --------------------------------------------------------------- | --------------------------------------------------------------------------------- |
+| <PyObject object="PartitionsDefinition" />                      | Superclass - defines the set of partitions that can be materialized for an asset. |
+| <PyObject object="HourlyPartitionsDefinition" />                | A partitions definition with a partition for each hour.                           |
+| <PyObject object="DailyPartitionsDefinition" />                 | A partitions definition with a partition for each day.                            |
+| <PyObject object="WeeklyPartitionsDefinition" />                | A partitions definition with a partition for each week.                           |
+| <PyObject object="MonthlyPartitionsDefinition" />               | A partitions definition with a partition for each month.                          |
+| <PyObject object="StaticPartitionsDefinition" />                | A partitions definition with a fixed set of partitions.                           |
+| <PyObject object="MultiPartitionsDefinition" />                 | A partitions definition with multiple dimensions.                                 |
+| <PyObject object="MultiPartitionKey" />                         | A multi-dimensional partition key.                                                |
+| <PyObject object="DynamicPartitionsDefinition" />               | A partitions definition whose partitions can be dynamically added and removed.    |
+| <PyObject object="OpExecutionContext" method="partition_key" /> | The partition key for the current run, which can be accessed in the computation   |
```

Comment:
```suggestion
| <PyObject object="OpExecutionContext" method="partition_key" /> | The partition key for the current run, which can be accessed in the computation.   |
```

Sorry for the nitpick - this makes it consistent with the rest of the descriptions